build/workers/pjs/jshint-worker.js

```diff
@@ -1,32 +0,0 @@
-var init = false;
```

Comment:
removed all files in build/workers/pjs because all of these files are being packaged in deps.json